### PR TITLE
01 Create List component with hardcoded data (#2)

### PR DESCRIPTION
**Pull request by ebarriosara**
01 Hardcoded list component

- Add component to show a member in a row.
- Add component to show a table with a row per member.
- Change the page component to show a hardcoded list of members.
- Define the view model.
- New readme.md.

### CODE REVIEW

File: 01 Hardcoded list component/src/pages/members/components/memberTable.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  memberList : MemberEntity[];
+}
+
+export const MemberTable = (props : Props) => (
+  <table className="table">
+    <thead>
+      <tr>
+        <th>Foto</th>
+        <th>Id</th>
+        <th>Nombre</th>
```
Comment (brauliodiez) [inline 01 Hardcoded list component/src/pages/members/components/memberTable.tsx]: One thing I forgot for this sample.let's takr it all im english

**Review by brauliodiez — COMMENTED** [review]
Good stuff, not sure if it would be better to change "photo" by "picture" but it's a minor thing, please go ahead merging implementation to start then asking PR from start to main repo master

**Comment by brauliodiez** [comment]
Good stuff !!

**Comment by brauliodiez** [comment]
Please proceed with the merge (I have not rights in your repo), then raise a new PR from start to the master branch in the original repo.

**Comment by ebarriosara** [comment]
Thanks!

I've renamed photo to picture as well.

**Pull request merged**